components/dashboard/src/Menu.tsx

```diff
@@ -180,55 +186,45 @@ export default function Menu() {
             ];
         }
         // Team menu
-        if (team) {
-            const currentUserInTeam = (teamMembers[team.id] || []).find((m) => m.userId === user?.id);
-
-            const teamSettingsList = [
-                {
-                    title: "Projects",
-                    link: `/t/${team.slug}/projects`,
-                    alternatives: [] as string[],
-                },
-                {
-                    title: "Members",
-                    link: `/t/${team.slug}/members`,
-                },
-            ];
-            if (showUsageView || (teamBillingMode && teamBillingMode.mode === "usage-based")) {
-                teamSettingsList.push({
-                    title: "Usage",
-                    link: `/t/${team.slug}/usage`,
-                });
-            }
-            if (currentUserInTeam?.role === "owner") {
-                teamSettingsList.push({
-                    title: "Settings",
-                    link: `/t/${team.slug}/settings`,
-                    alternatives: getTeamSettingsMenu({ team, billingMode: teamBillingMode }).flatMap((e) => e.link),
-                });
-            }
-
-            return teamSettingsList;
+        if (!team) {
+            return [];
         }
-        // User menu
-        const userMenu = [];
-        userMenu.push({
-            title: "Projects",
-            link: "/projects",
-        });
-        if (showUsageView) {
-            userMenu.push({
+        const currentUserInTeam = (teamMembers[team.id] || []).find((m) => m.userId === user?.id);
+
+        const teamSettingsList = [
+            {
+                title: "Projects",
+                link: `/t/${team.slug}/projects`,
+                alternatives: [] as string[],
+            },
+            {
+                title: "Members",
+                link: `/t/${team.slug}/members`,
+            },
+        ];
+        if (showUsageView || (teamBillingMode && teamBillingMode.mode === "usage-based")) {
+            teamSettingsList.push({
                 title: "Usage",
-                link: "/usage",
+                link: `/t/${team.slug}/usage`,
             });
         }
-        userMenu.push({
-            title: "Settings",
-            link: "/settings",
-            alternatives: getSettingsMenu({ userBillingMode }).flatMap((e) => e.link),
-        });
-        return userMenu;
+        if (currentUserInTeam?.role === "owner") {
+            teamSettingsList.push({
+                title: "Settings",
+                link: `/t/${team.slug}/settings`,
+                alternatives: getTeamSettingsMenu({ team, billingMode: teamBillingMode }).flatMap((e) => e.link),
+            });
+        }
+
+        return teamSettingsList;
     })();
+    const leftMenu: Entry[] = [
+        {
+            title: "Workspaces",
```

Comment:
**issue(non-blocking)**: Mixing three level of navigation elements in the same dense area still seems challenging, but let's merge this and move it from there. 🛹 

<img width="657" alt="Frame 1115" src="https://user-images.githubusercontent.com/120486/193855356-0b7cf8d6-be6d-46b7-a29f-1107dd98d5ef.png">